pke/kyber/internal/common/field_test.go

```diff
@@ -35,9 +36,29 @@ func TestBarrettReduceFull(t *testing.T) {
 	}
 }
 
+func randSliceUint32(N uint) []uint32 {
```

Comment:
It seems like most everywhere this function is used there is an effective max. Why not add the max as a parameter here?

Reply:
good one!